docs/knn/index.md

```diff
@@ -125,6 +125,34 @@ GET my-knn-index-1/_search
 }
 ```
 
+The formulas we use to perform calculations are as follows:
+
+Euclidian Distance:
+<p>
+
+\[ Distance(X, Y) = \sqrt{\sum_{i=1}^n (X_i - Y_i)^2} \]
+
+</p>
+
+Cosine similarity:
+<p>
+  \[ {A &middot; B \over \|A\| &middot; \|B\|} =
+
+  {\sum_{i=1}^n (A_i &middot; B_i) \over \sqrt{\sum_{i=1}^n A_i^2} &middot; \sqrt{\sum_{i=1}^n B_i^2}}\]
+ where \(\|A\|\) and \(\|B\|\) represent normalized vectors.
+
+</p>
+
+After calculations, we perform the following conversions to get the final Elasticsearch score. Note that these conversions are how we calculate approximate k-NN scores. For custom scoring conversions, see the [custom scoring](#Pre-filtering-with-custom-scoring) section.
```

Comment:
Typically, we'd use "we" for "we, the humans who run this service" or "we, the maintainers of ODFE." In this case, the KNN plugin is performing the conversion, so we want to say something like, "After base calculations, the k-NN plugin performs the following conversions to get the final Elasticsearch relevance score. Note that these conversions are for approximate k-NN scores. For custom scoring conversions, see..."